src/reporter/tests/test_attribute_name_case.py

```diff
@@ -0,0 +1,140 @@
+from conftest import QL_URL, do_clean_crate
+import pytest
+import requests
+import time
+import urllib
+from .utils import send_notifications
+
+
+entity_type = 'TestDevice'
+
+attr1 = 'AtTr1'
+attr2 = 'aTtr_2'
+
+attr1_value = '1'
+attr2_value = 2
+
+entity1_id = 'd1'
+entity2_id = 'd2'
+
+
+def mk_entity(eid):
+    return {
+        'id': eid,
+        'type': entity_type,
+        attr1: {
+            'type': 'Text',
+            'value': attr1_value
+        },
+        attr2: {
+            'type': 'Number',
+            'value': attr2_value
+        }
+    }
+
+
+def mk_entities():
+    return [
+        mk_entity(entity1_id), mk_entity(entity1_id),
+        mk_entity(entity2_id), mk_entity(entity2_id)
+    ]
+
+
+def insert_entities():
+    notification_data = [{'data': mk_entities()}]
+    send_notifications(notification_data)
+
+
+@pytest.fixture(scope='module')
+def manage_db_entities():
```

Comment:
as we agreed earlier in the call we'll have to hit the REST API in those tests, so we'll have to keep this method the way it is. but thanks for telling me about the translator fixture, I'll definitely keep this in mind going forward since I can see it can come in handy in many situations!